docs/BindingsGuide_DotnetOutOfProc.md

```diff
@@ -0,0 +1,187 @@
+# SQL Extension .NET Worker
+
+Welcome to the Sql Extension .NET Worker Repository. The .NET Worker provides .NET 6 support for SQL Bindings in Azure Functions, introducing an **Isolated Model**, running as an out-of-process language worker that is separate from the Azure Functions runtime. This allows you to have full control over your application's dependencies as well as other new features like a middleware pipeline. A .NET Isolated function app works differently than a .NET Core 3.1 function app. For .NET Isolated, you build an executable that imports the .NET Isolated language worker as a NuGet package. Your app includes a [`Program.cs`](https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-outofproc/Program.cs) that starts the worker.
```

Comment:
You can move this blurb back into a README in the out of proc directory. This guide should be specifically for creating functions with SQL bindings in the different languages - where as this stuff is more about the actual worker itself. 

(same goes for the other content in here that isn't relevant to creating a function)